com.visualstudio.code.tool.docker.metainfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>com.visualstudio.code.tool.docker</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>Podman</name>
+  <summary>Docker is a platform built for developers to build and run applications.</summary>
+  <url type="homepage">https://docker.com/</url>
+  <releases>
+    <release version="20.10.12" date="2022-03-29"/>
```

Comment:
shouldn't date be "2020-10-12"

Reply:
So it should be the release date of the upstream software I guess? Makes sense, will change

Reply:
The actual release date of 20.10.12 seems to be 2021-12-13, if I'm reading the release notes correctly. Gonna use that. (Don't ask me how that naming convention works^^')